pkg/postgres/repository.go

```diff
@@ -327,6 +329,87 @@ WHERE most_recent_event_moment < $1`,
 	return
 }
 
+func (a *RepositoryAgent) GetRecentlyActiveRuns(touchThreshold EasternTime) (runs map[martaapi.Line]map[martaapi.Direction][]Run, err error) {
```

Comment:
Collects all the data about any runs that have been updated since `touchThreshold`

Reply:
If this query is slow, we can select fewer columns and add some indices. Right now that doesn't appear necessary (it's taking about 30ms against the live data 🤷‍♀)

Reply:
nother nit: would it be worth adding your first comment as a doc comment for the function?